plugins/modules/iosxr_lldp_interfaces.py

```diff
@@ -36,12 +36,12 @@
     "supported_by": "network",
 }
 
-DOCUMENTATION = """module: iosxr_lldp_interfaces
-short_description: Manage Link Layer Discovery Protocol (LLDP) attributes of interfaces
-  on IOS-XR devices.
+DOCUMENTATION = """module: cisco.iosxr.iosxr_lldp_interfaces
```

Comment:
would this be cisco.iosxr.iosxr_lldp_interfaces of just iosxr_lldp_interfaces  ?